tests/integration/targets/homebrew/tasks/main.yml

```diff
@@ -0,0 +1,57 @@
+# Test code for the homebrew module.
+# Copyright: (c) 2020, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Use ignored-pinned option while upgrading all
```

Comment:
@mattclay Do you know how I can run this task with non-privileged user? Since brew is failing with root user.

Reply:
@Akasurde Yes, see example here:

https://github.com/ansible-collections/community.general/blob/f799aa8356825841cf940e971e1dc036bea76f75/tests/integration/targets/iso_extract/tasks/7zip.yml#L45-L66

Reply:
Thanks. I didn't see that :)